shared/domains/data/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+// =======================
+// Domain: Data persistence
+// =======================
+
+import _ from 'lodash'
```

Comment:
As far as I know, there is little reason to use lodash. Lodash is [on our list of dependencies to completely get rid of](https://github.com/okTurtles/group-income-simple/issues/361). The simple object `_store` should be enough? If not, maybe one of ES6's new collection types like `Map` or whatever.

Reply:
the only reason I chose lodash for this is its ability to consistently handle non-existent deep paths in the `_store` object. eg. easily create `_store.events.testEvent.listeners` while creating `_store.events` and `_store.events.testEvent` if they didn't exist yet, or, for the getters, returning an `undefined` instead of erroring at every non-existent level.

but I actually wrote some of that recursive function to handle this before throwing it out for the sake of simplification with lodash, so it won't be hard to reproduce 👍 but I will check the new collection types as well.